apiservers/engine/CLI-COMPATIBILITY.md

```diff
@@ -0,0 +1,274 @@
+# Docker CLI Compatibility
+VIC attempts to be compatible with Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.
+
+The list is organized into 3 separate tables.  The first list CLI operations that is supported in the first version of VIC.  The second list are operations that VIC may support in the future.
+
+### Supported in VIC 1.0
+
+|Docker CLI Operation | VIC Supported?|
+|--- | ---|
+|attach | YES|
+|cp | YES|
+|create | *Yes - differences*|
+|events | *Yes - differences*|
+|history | YES|
+|images | YES|
+|login | YES|
+|logout | YES|
+|logs | YES|
+|network connect| YES|
+|network create| *Yes - differences*|
+|network disconnect| YES|
+|network ls| YES|
+|network rm| YES|
+|port | YES|
+|ps | YES|
+|pull | YES|
+|rm | YES|
+|rmi | YES|
+|run | *Yes - differences*|
+|rename | YES|
+|restart | YES|
+|start | YES|
+|stop | YES|
+|tag| YES|
+|version | YES|
+|volume create| *Yes - differences*|
+|volume inspect| YES|
+|volume ls| YES|
+|volume rm| YES|
+
+### Support not Guaranteed
+
+|Docker CLI Operation |
+|--- |
+|exec |
+|pause |
+|stats |
+|top |
+|unpause |
+|wait |
+
+### Future Support
+
+|Docker CLI Operation |
+|--- |
+|build |
+|commit |
+|diff |
+|export |
+|import |
+|load |
+|push |
+|save |
+|search |
+
+The following sections dive further into these differences.
+
+## create
+
+```
+Usage: docker create [OPTIONS] IMAGE [COMMAND] [ARG...]
+
+Create a new container
+```
+
+| params | Docker | VIC |
+|---|---|---|
+|-a, --attach=[]|Attach to STDIN, STDOUT or STDERR|YES|
+|--add-host=[] |Add a custom host-to-IP mapping (host:ip)|YES|
+|-c, --cpu-shares=0 | CPU shares (relative weight)|*diff*|
+|--cap-add=[]|Add Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cap-drop=[]|Drop Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cgroup-parent=""|Optional parent cgroup for the container|NO, VIC containers do not run in cgroups|
```

Comment:
--cgroup-parent=""|Optional parent cgroup for the container|VIC containers do not run in cgroups|